src/olympia/addons/tasks.py

```diff
@@ -358,3 +358,36 @@ def resize_icon(source, dest_folder, target_sizes, **kw):
         return {'icon_hash': icon_hash}
     except Exception as e:
         log.error(f'Error saving addon icon ({dest_file}): {e}')
+
+
+@task
+@set_modified_on
+def resize_preview(src, preview_pk, **kw):
+    """Resizes preview images and stores the sizes on the preview."""
+    preview = Preview.objects.get(pk=preview_pk)
+    preview.sizes = {'thumbnail_format': amo.ADDON_PREVIEW_SIZES['thumbnail_format']}
+    thumb_dst, full_dst, orig_dst = (
+        preview.thumbnail_path,
+        preview.image_path,
+        preview.original_path,
+    )
+    log.info('[1@None] Resizing preview and storing size: %s' % thumb_dst)
```

Comment:
Just curious, what is this `[1@None]`?

Reply:
I just moved this function from devhub/tasks.py wholesale, but I believe it's to match the logging format of tasks that work on multiple object instances... not that I could find any other logging for tasks that worked with multiple objects that was a matching format.  @diox can you remember anything more?

Reply:
thanks @diox.  It doesn't seem like any priority to change these log statements so I'm not going to file a follow-up to change them.